test_agent/python/testagent.py

```diff
@@ -344,6 +351,93 @@ def handle_uuid_validate_command(json_msg):
     )
 
 
+def handle_uattributes_validate_command(json_msg: Dict[str, Any]):
```

Comment:
Can this function be implemented in tck step file. Test agent are just forwarding json msg all the validation happens at test manager.

Reply:
This may not be possible if we need to test specific to each language. In that case this comment is invalid